packages/components/pie-thumbnail/src/defs.ts

```diff
@@ -1,3 +1,21 @@
-// TODO - please remove the eslint disable comment below when you add props to this interface
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface ThumbnailProps {}
+import type { ComponentDefaultProps } from '@justeattakeaway/pie-webc-core';
+
+export const variants = [
+    'default', 'outline',
+] as const;
+
+export type Variant = typeof variants[number];
+
+export interface ThumbnailProps {
+    variant?: Variant;
+    src?: string;
+    alt?: string;
+}
+
+export type DefaultProps = ComponentDefaultProps<ThumbnailProps, 'variant' | 'src' | 'alt'>;
+
+export const defaultProps: DefaultProps = {
+    variant: 'default',
+    src: './assets/placeholder.png',
+    alt: 'placeholder',
```

Comment:
I think we shouldn't assign default values here, as we don't want to make any assumptions about end users and let them decide to set what they see fit.

The placeholder behavior when the src isn't available will be handled in a separate ticket, and that will also be a prop.

```suggestion
    src: '',
    alt: '',
```

Reply:
I was wondering what would happen if the end user doesn't pass src and doesn't provide a placeholder - what would the expected behaviour be then? 

In any case I agree that this is an issue for another task, I will leave the defaults be empty strings as you suggest.

Reply:
users will need to pass the placeholder props to provide a placeholder.
if they don't pass a placeholder and the src is empty, then simply the img won't render similar to how the img tag is in the HTML specs.